app/javascript/gobierto_data/webapp/components/sets/data/SQLEditorCode.vue

```diff
@@ -122,7 +150,11 @@ export default {
 
         // update the query while typing
         this.$root.$emit("setCurrentQuery", value);
-      }, 250);
+        this.$root.$emit('enableSavedButton')
+        this.$root.$emit("resetQuery", false);
+        this.$root.$emit("revertSavedQuery", false);
+        this.$root.$emit("disabledStringSavedQuery", false);
```

Comment:
Move this logic, perhaps the events become useless, to the Dataset component, since it's it who handles the logic